clang/lib/Driver/Driver.cpp

```diff
@@ -4342,10 +4339,8 @@ class OffloadingActionBuilder final {
         ActionList WrapperInputs;
         // post link is not optional - even if not splitting, always need to
         // process specialization constants
-        bool MultiFileActionDeps = !isSpirvAOT || DeviceCodeSplit || EnableDAE;
-        types::ID PostLinkOutType = isNVPTX || !MultiFileActionDeps
-                                        ? types::TY_LLVM_BC
-                                        : types::TY_Tempfiletable;
+        types::ID PostLinkOutType =
```

Comment:
please update the [picture and the doc](https://github.com/intel/llvm/blob/sycl/sycl/doc/CompilerAndRuntimeDesign.md#device-code-post-link-step)